CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v1.18.0
+
+<a name="breaking_changes_1.18.0">[Breaking Changes:](#breaking_changes_1.18.0)</a>
+
+- [core] `setTopPanelVisibily` renamed to `setTopPanelVisibility`
```

Comment:
Almost there :) We want to add a link to the corresponding pull-request so adopters can easily view the changes in question:

```suggestion
- [core] `setTopPanelVisibily` renamed to `setTopPanelVisibility` [#10020](https://github.com/eclipse-theia/theia/pull/10020)
```